daffodil-core/src/main/scala/org/apache/daffodil/dsom/Term.scala

```diff
@@ -107,10 +108,43 @@ trait Term
     statements.foreach { _.checkTerm(this) }
   }
 
+  // Used to recursively go through Terms and look for DFDL properties that
```

Comment:
Should be scaladoc, not just a // comment.

Reply:
WIll fix.